doc/source/conf.py

```diff
@@ -193,7 +194,7 @@ def reset_servers(gallery_conf, fname, when):
 # -- Options for HTML output -------------------------------------------------
 html_short_title = html_title = "PyDPF-Core"
 html_theme = "ansys_sphinx_theme"
-html_logo = pyansys_logo_black
+html_logo = pyansys_logo_light_mode
```

Comment:
@luisaFelixSalles why do we need this change? Is it going to work in dark mode?

Reply:
The current nav bar looks like this:

![image](https://github.com/user-attachments/assets/7bfe6c8e-3b24-4feb-8b8c-9dfa397b9f21)

 The logo has a black square background, but we dont see it because the navigation bar also has a black background.

So, I changed the logo for the one with a light back ground, to match the new navigation bar color. It actually doesn't look good in dark mode, I'll look into other projects to see how they do it

Reply:
@Revathyvenugopal162 I would like to ask for your help also in this case:

I tried passing the logo as an html configuration option with:
![image](https://github.com/user-attachments/assets/5208596f-d939-4b49-a2c1-8d15711e8cd3)

But I get the following error when building the documentation: 

![image](https://github.com/user-attachments/assets/d7f6683a-43d1-4e30-929b-07dd22a6db10)

Do you know if I'm missing other configuration changes in other to solve this problem?

Reply:
This wont work with out new theme. Please add this "logo": "pyansys" in theme options. example PR , https://github.com/ansys/pyansys-geometry/commit/40366c1dfbdc13664d9c9448a970819a0990978e#diff-d8d3ed25802824d15bf411f8e97416d8fc6f9247e821b0617f2b869dc584b99c

Reply:
I tried this but it gives me the same error. 
Is there a way to implement the logo (that respects the light and dark modes) with the new theme?

Reply:
@luisaFelixSalles @PProfizi The logo issue was fixed in [33fade6](https://github.com/ansys/pydpf-core/pull/1749/commits/33fade68ec366d400711f080eceec6f563fd789d) and this is the first time we are getting this issue. i will keep track on this and try to fix it in ansys sphinx theme if required

Reply:
Thank you for your help ! 🙂